bikeshed/__init__.py

```diff
@@ -2551,8 +2558,15 @@ def inlineRemoteIssues(doc):
                 if el.tag == "p":
                     el.tag = "div"
                 addClass(el, "no-marker")
-        except:
-            pass
+        except urllib2.HTTPError as err:
+            if doc.token and err.code == 401:
+                die("Unauthorized Access to GitHub's API. There might be an issue with your token.")
```

Comment:
die() *does* terminate the process *by default*, but it's common and expected to sometimes run bikeshed with the `--force` option, which prevents that.  I try my best to recover gracefully from any fatal errors.

`return` is too strong here; you only need to exit the loop.  If people are embedding inline issues from multiple types of repositories, for example (which isn't supported now, but could be in the future), the GH issues failing shouldn't prevent any following types of issues from working.

I think the 401 and 403 errors should use `break`, and the other two probably just a `continue`, in case it's a transient issue that further requests won't suffer from.